frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,213 @@
+});
+
+// sudebar submenu groupings
+const LinkListItem = styled(ListItem)({
+  backgroundColor: "#FFFFFF",
+  "&.Mui-selected": {
+    backgroundColor: "#EBEDFB",
+    ".MuiTypography-root": {
+      color: "#3548D4",
+    },
+  },
+  "&:hover": {
+    backgroundColor: "#E7E7EA",
+  },
+  height: "48px",
+});
+
+const LinkHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+});
 
 interface GroupProps {
   heading: string;
   open: boolean;
   updateOpenGroup: (heading: string) => void;
-  onNavigate: () => void;
 }
 
-const Group: React.FC<GroupProps> = ({
-  heading,
-  open = false,
-  updateOpenGroup,
-  onNavigate,
-  children,
-}) => {
+const Group: React.FC<GroupProps> = ({ heading, open = false, updateOpenGroup, children }) => {
   const childrenList = React.Children.toArray(children);
 
+  const [openList, setListOpen] = React.useState(false);
+
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setListOpen(!openList);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setListOpen(false);
+  };
+
+  let headingPath = window.location.pathname.replace("/", "").split("/")[0];
+
+  if (headingPath === "ec2") {
+    headingPath = "aws";
+  }
+
+  const isSelected = headingPath == heading.toLowerCase();
+
   // n.b. if a Workflow Grouping has no workflows in it don't display it even if
   // it's not explicitly marked as hidden.
   if (childrenList.length === 0) {
     return null;
   }
+
   return (
-    <List data-qa="workflowGroup">
-      <ListItem button onClick={() => updateOpenGroup(heading)}>
-        <GroupHeading color="primary" variant="h6">
-          {heading}
-        </GroupHeading>
-        <Grid container justify="flex-end" data-qa="toggle">
-          {open ? <ExpandLessIcon /> : <ExpandMoreIcon />}
-        </Grid>
-      </ListItem>
-      {!open ? <Divider variant="middle" /> : null}
-      <Collapse in={open} timeout="auto" unmountOnExit>
-        <List component="div" disablePadding>
-          {childrenList.map((c: React.ReactElement) => {
-            return React.cloneElement(c, { onClick: onNavigate });
-          })}
-        </List>
-      </Collapse>
-    </List>
+    <GroupList data-qa="workflowGroup">
+      <GroupListItem
+        button
+        selected={isSelected}
+        ref={anchorRef}
+        aria-controls={openList ? "workflow-options" : undefined}
+        aria-haspopup="true"
+        onClick={() => {
+          handleToggle();
+          updateOpenGroup(heading);
+        }}
+      >
+        <Avatar>{heading.charAt(0)}</Avatar>
+        <GroupHeading align="center">{heading}</GroupHeading>
+        <Collapse in={open} timeout="auto" unmountOnExit>
+          <Popper
+            open={openList}
+            anchorEl={anchorRef.current}
+            role={undefined}
```

Comment:
nit: does this have any effect or purpose?
```suggestion
```

Reply:
removed. saw this is on a material ui ex & misunderstood it as a prop needed for popper functionality

i looked it up and it's an accessibility attribute for screen readers